lib/rust/parser/debug/tests/parse.rs

```diff
@@ -1230,6 +1248,13 @@ fn attributes() {
     test!("@a\n@b\nx", (Annotated "@" a () #(()) (Annotated "@" b () #(()) (Ident x))));
 }
 
+#[test]
+fn attributes_in_types() {
+    test!("type A\n @a z\n @b\n x",
+        (TypeDef type A #() #(
+         ((Binding (Annotated "@" a (Ident z) #(()) (Annotated "@" b () #(()) (Ident x))))))));
```

Comment:
This is an example similar to code @4e6 reported as being without `Annotated`. Now it seems to be annotated.